Auto update failing for poac

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [poac](https://github.com/termux/termux-packages/blob/1d150b5e5dd61a69f543c772d049e927298a9378/packages/poac) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating poac [Current version: 0.9.3]
INFO: package being updated to 0.10.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  158k    0  158k    0     0   235k      0 --:--:-- --:--:-- --:--:--  235k

You are about to commit these changes:

--------------------
diff --git a/packages/poac/build.sh b/packages/poac/build.sh
index 52acd758e..6934ae4a4 100644
--- a/packages/poac/build.sh
+++ b/packages/poac/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/poac-dev/poac
 TERMUX_PKG_DESCRIPTION="A package manager and build system for C++"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.9.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="0.10.0"
 TERMUX_PKG_SRCURL="https://github.com/poac-dev/poac/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=122aa46923e3e93235305b726617df7df747ed7a26072ccd6b87ffaf84a33aed
+TERMUX_PKG_SHA256=4bdede67b28f9622c071bef8c7eae76062c9ef2ad122deee49d994668e846288
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_BUILD_DEPENDS="nlohmann-json"
--------------------

bump(main/poac): 0.10.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package poac exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building poac for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0   171k      0 --:--:-- --:--:-- --:--:--  173k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3557      0 --:--:-- --:--:-- --:--:--  3544
gpg: Signature made Fri 09 Aug 2024 12:42:36 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13319  100 13319    0     0  58280      0 --:--:-- --:--:-- --:--:-- 58416
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3393      0 --:--:-- --:--:-- --:--:--  3400
gpg: Signature made Thu 08 Aug 2024 06:33:29 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  55376      0 --:--:-- --:--:-- --:--:-- 55354
100 13119  100 13119    0     0  55349      0 --:--:-- --:--:-- --:--:-- 55354
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   3575      0 --:--:-- --:--:-- --:--:--  3590
gpg: Signature made Fri 09 Aug 2024 12:51:50 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libnghttp2@1.62.1 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp2/libnghttp2_1.62.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 99516  100 99516    0     0  1073k      0 --:--:-- --:--:-- --:--:-- 1079k
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp3@1.4.0 if necessary...
Found libnghttp3 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnghttp3/libnghttp3_1.4.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 64044  100 64044    0     0   766k      0 --:--:-- --:--:-- --:--:--  772k
extracting libnghttp3 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.07.02 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates/ca-certificates_1:2024.07.02_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  122k  100  122k    0     0  1322k      0 --:--:-- --:--:-- --:--:-- 1329k
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.07.02 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/ca-certificates-java/ca-certificates-java_1:2024.07.02_all.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  111k  100  111k    0     0  1339k      0 --:--:-- --:--:-- --:--:-- 1347k
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.3.1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl/openssl_1:3.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 2266k  100 2266k    0     0  18.2M      0 --:--:-- --:--:-- --:--:-- 18.2M
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/o/openssl-tool/openssl-tool_1:3.3.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  256k  100  256k    0     0  2627k      0 --:--:-- --:--:-- --:--:-- 2643k
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libssh2@1.11.0 if necessary...
Found libssh2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libs/libssh2/libssh2_1.11.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  193k  100  193k    0     0  2090k      0 --:--:-- --:--:-- --:--:-- 2099k
extracting libssh2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcurl@8.9.1 if necessary...
Found libcurl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libcurl/libcurl_8.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  869k  100  869k    0     0  6584k      0 --:--:-- --:--:-- --:--:-- 6590k
extracting libcurl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency curl@8.9.1 if necessary...
Found curl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/c/curl/curl_8.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  203k  100  203k    0     0  2043k      0 --:--:-- --:--:-- --:--:-- 2051k
extracting curl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  4861k      0 --:--:-- --:--:-- --:--:-- 4851k
100  543k  100  543k    0     0  4857k      0 --:--:-- --:--:-- --:--:-- 4851k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0  16698      0 --:--:-- --:--:-- --:--:-- 16706
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@27 if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libc/libc++/libc++_27_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 10  291k   10 30614    0     0  34569      0  0:00:08 --:--:--  0:00:08 34553
100  291k  100  291k    0     0   242k      0  0:00:01  0:00:01 --:--:--  242k
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre@8.45-1 if necessary...
Found pcre in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre/pcre_8.45-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  635k  100  635k    0     0  6429k      0 --:--:-- --:--:-- --:--:-- 6483k
extracting pcre to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcregrep@8.45-1 if necessary...
Found pcregrep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcregrep/pcregrep_8.45-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 24588  100 24588    0     0  32917      0 --:--:-- --:--:-- --:--:-- 32915
100 24588  100 24588    0     0  32912      0 --:--:-- --:--:-- --:--:-- 32915
extracting pcregrep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpcreposix@8.45-1 if necessary...
Found libpcreposix in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libp/libpcreposix/libpcreposix_8.45-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  3328  100  3328    0     0   5997      0 --:--:-- --:--:-- --:--:--  6007
extracting libpcreposix to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.44 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2/pcre2_10.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  868k  100  868k    0     0  8061k      0 --:--:-- --:--:-- --:--:-- 8113k
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.44 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2grep/pcre2grep_10.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
100 33456  100 33456    0     0  10014      0  0:00:03  0:00:03 --:--:-- 10016
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgit2@1.8.1 if necessary...
Found libgit2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgit2/libgit2_1.8.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  674k  100  674k    0     0  6241k      0 --:--:-- --:--:-- --:--:-- 6247k
extracting libgit2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libtbb@2021.13.0 if necessary...
Found libtbb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libt/libtbb/libtbb_2021.13.0_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
 10  273k   10 30612    0     0   7057      0  0:00:39  0:00:04  0:00:35  7056
 34  273k   34 96148    0     0  15666      0  0:00:17  0:00:06  0:00:11 19486
 69  273k   69  189k    0     0  27200      0  0:00:10  0:00:07  0:00:03 39330
100  273k  100  273k    0     0  38834      0  0:00:07  0:00:07 --:--:-- 69952
extracting libtbb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency nlohmann-json@3.11.3 if necessary...
Found nlohmann-json in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/nlohmann-json/nlohmann-json_3.11.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
100  107k  100  107k    0     0  57203      0  0:00:01  0:00:01 --:--:-- 57193
extracting nlohmann-json to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/poac-dev/poac/archive/refs/tags/0.10.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  158k  100  158k    0     0   248k      0 --:--:-- --:--:-- --:--:--  248k
Applying patch: src-Git2-Exception.cc.patch
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
Package fmt was not found in the pkg-config search path.
Perhaps you should add the directory containing `fmt.pc'
to the PKG_CONFIG_PATH environment variable
No package 'fmt' found
Package fmt was not found in the pkg-config search path.
Perhaps you should add the directory containing `fmt.pc'
to the PKG_CONFIG_PATH environment variable
No package 'fmt' found
Package fmt was not found in the pkg-config search path.
Perhaps you should add the directory containing `fmt.pc'
to the PKG_CONFIG_PATH environment variable
No package 'fmt' found
Package fmt was not found in the pkg-config search path.
Perhaps you should add the directory containing `fmt.pc'
to the PKG_CONFIG_PATH environment variable
No package 'fmt' found
git clone https://github.com/ToruNiina/toml11.git build-out/DEPS/toml11
Cloning into 'build-out/DEPS/toml11'...
Error: fmt >= 8.1.1, fmt < 11.0.0 not found
make: *** [Makefile:68: check_deps] Error 1
make: *** Waiting for unfinished jobs....
git -C build-out/DEPS/toml11 reset --hard v4.0.3
HEAD is now at 5ac47b8 chore: bump version to 4.0.3
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-08-09 06:15:38 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/10314483850">10314483850</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/poac/build.sh
+++ b/packages/poac/build.sh
@@ -2,15 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://github.com/poac-dev/poac
 TERMUX_PKG_DESCRIPTION="A package manager and build system for C++"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="0.9.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="0.10.0"
 TERMUX_PKG_SRCURL="https://github.com/poac-dev/poac/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz"
-TERMUX_PKG_SHA256=122aa46923e3e93235305b726617df7df747ed7a26072ccd6b87ffaf84a33aed
+TERMUX_PKG_SHA256=4bdede67b28f9622c071bef8c7eae76062c9ef2ad122deee49d994668e846288
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_BUILD_DEPENDS="nlohmann-json"
-TERMUX_PKG_DEPENDS="libtbb, libgit2, libcurl"
-TERMUX_PKG_SUGGESTS="clang, make, pkg-config, fmt"
+TERMUX_PKG_BUILD_DEPENDS="fmt, nlohmann-json"
+TERMUX_PKG_DEPENDS="libc++, libcurl, libgit2, libtbb"
+TERMUX_PKG_SUGGESTS="clang, make, pkg-config"
 TERMUX_PKG_BLACKLISTED_ARCHES="arm, i686"
 
 termux_step_make() {

